stable/mongodb-replicaset/init/add_to_rs.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This script parses the optional replica-config.js
+ * to find the replica config for a particular member
+ * of the replicaset.
+ *
+ * It then adds the member, along with their configuration,
+ * to the replica.
+ */
+load('/init/replica-config.js');
+
+var memberIndex = xxMEMBER_INDEXxx;
+var memberConfig = setConfig ? setConfig[memberIndex] : {};
+memberConfig.host = "xxMEMBER_HOSTxx";
+
+if (rs.status().codeName === "NotYetInitialized") {
```

Comment:
I'm not a fan of templating in js either. I think you should revert to how it was with no js, and use jq in bash to extract the particular replicaset config member you need. It seems jq is already in the mongodb image.

Reply:
🙌 didn't realize `jq` was on there!